managed/services/telemetry/config.default.yml

```diff
@@ -67,57 +59,189 @@ telemetry:
       - metric_name: "mongodb_version"
         label: "mongodb"
 
+  - id: PMMPostgresDBVersion
+    source: VM
+    query: pg_static{service_name="pmm-server-postgresql", service_type="postgresql"}
+    summary: "Monitored PostgreSQL services version"
+    data:
+      - metric_name: "pmm_server_postgresql_version"
+        label: "short_version"
+
   - id: PostgresDBVersion
     source: VM
-    query: pg_static{service_type="postgresql"}
+    query: pg_static{service_name!="pmm-server-postgresql", service_type="postgresql"}
     summary: "Monitored PostgreSQL services version"
     data:
       - metric_name: "postgresql_version"
         label: "short_version"
 
-#
+  #
   - id: GrafanaStatTotalUsers
     source: VM
     query: grafana_stat_total_users
     summary: "Total Grafana users"
     data:
-      - metric_name: "grafana_stat_total_users"
+      - metric_name: "pmm_server_grafana_stat_total_users"
         value: 1
 
-# PMM - Usage
+  # PMM - Usage
   - id: PMMUsageNodesCount
     source: PMMDB_SELECT
     query: count(*) AS nodes_count FROM nodes
     summary: "Monitored nodes count"
     data:
-      - metric_name: "pmm_usage_nodes_count"
+      - metric_name: "pmm_server_usage_nodes_count"
         column: "nodes_count"
 
   - id: PMMUsageServicesCount
     source: PMMDB_SELECT
     query: count(*) AS services_count FROM services
     summary: "Monitored services count"
     data:
-      - metric_name: "pmm_usage_services_count"
+      - metric_name: "pmm_server_usage_services_count"
         column: "services_count"
 
-# Nodes information
-  - id: NodesInfoAgentsVersions
+  # Nodes information
+  - id: NodesInfoPMMAgentEmbedded
```

Comment:
it  should, but .. as we found using PG  version - this can be our own bugs too .. so this can help

Reply:
but we release PMM Server together with PMM Client, so in case we need to know how many PMM Clients of particular version we have - just count PMM Servers of this version.

Reply:
in theory, people can replace binaries of exporters  .. so this should cover this case. 
Also, it's not very convenient to "calculate" version of exporter  by pmm-agent  version